package.json

```diff
@@ -30,5 +31,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "prettier": "^2.1.1"
```

Comment:
Pin exact version. Prettier formatting changes in patch versions.

Reply:
Ok, done.